docs/faq.md

```diff
@@ -54,6 +54,21 @@ scpca_sample = anndata.readh5ad(file = "SCPCL000000_processed_rna.h5ad")
 A full description of the contents of the `AnnData` object can be found in the section on {ref}`Components of an AnnData object <sce_file_contents:Components of an anndata object>`.
 For more information on working with the H5AD files, see {ref}`Getting started with an ScPCA dataset <getting_started:Getting started with an scpca dataset>`.
 
+## Which samples can I download as AnnData objects?
+
+Most samples in the ScPCA Portal are available for download as both `SingleCellExperiment` objects (`.rds` files) and `AnnData` objects (`.h5ad` files).
+There are two types of samples where `AnnData` objects are not available:
+
+- Spatial transcriptomics data from a given sample
+    - As described in {ref}`the spatial transcriptomics processing section<processing_information:spatial transcriptomics>`, no post-processing is performed on these libraries after running Space Ranger.
+    Therefore, we only provide the output from running Space Ranger and do not store data in either R or Python objects.
+
+- Samples that are part of multiplexed libraries
+    - Although the ScPCA pipeline {ref}`reports demultiplexing results<processing_information:HTO demultiplexing>`, it does not actually perform demultiplexing.
+    `AnnData` objects do not support storage of both gene expression data and additional modalities, like quantified HTO expression data, within the same object.
+    Without the associated HTO data, the sample of origin cannot be identified for each cell, causing ambiguity in sample origin assignments.
+    Therefore, we do not provide any multiplexed libraries as `AnnData` objects.
```

Comment:
To be honest, there probably isn't a good reason not to have AnnData for multiplexed libraries, though it would require us to figure out how to make tables of results. I think the main reason may be that anything we would produce would not be compatible with CellXGene? At least that is my recollection of the initial logic. Not doing the merged samples is a bit easier to explain, because of the challenge of mapping reused HTO tags, but that doesn't apply here. 

Here is my attempt:

```suggestion
    - Although the ScPCA pipeline {ref}`reports demultiplexing results<processing_information:HTO demultiplexing>`, it does not definitively separate samples due to the potential for disagreement among methods.
    - Resolving such disagreements requires examination of the HTO data, which can not be stored in the same `AnnData` object. 
    Therefore, we do not currently provide any multiplexed libraries as `AnnData` objects.
```

Reply:
> Yes, this is exactly why we don't have them which is why I was struggling. 

Do we maybe want to say this explicitly then?